scripts/c-btmInference.py

```diff
@@ -0,0 +1,122 @@
+from transformers import AutoModelForCausalLM, AutoTokenizer
+import torch
+from torch.nn import PairwiseDistance
+import numpy 
+
+def load_models(model_names):
+    """
+    takes in list of model names and loads model and corresponding tokenizers
+    returns separate lists of models and tokenizers to be accessed by other functions later
+    """
+    models = []
+    tokenizers = []
+
+    for model_name in model_names:
+        model = AutoModelForCausalLM.from_pretrained(model_name)

```

Comment:
some issues with loading the models and maintaining HF credentials -- had to load models and tokenizers outside of the function

Reply:
Ok. good to know. Strange that you can't load. I made all models public now